src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageItemViewModel.cs

```diff
@@ -79,7 +79,7 @@ public PackageItemViewModel(INuGetSearchService searchService, PackageModel pack
 
         public ImmutableList<KnownOwnerViewModel> KnownOwnerViewModels { get; internal set; }
 
-        public string Owner => string.Join(",", _packageModel.OwnersList ?? []);
+        public string Owner => string.Join(", ", _packageModel.OwnersList ?? []);
```

Comment:
Do we know if WPF only reads a property once?   Because if there's any chance that the property could be read more than once, it'd probably be good to have a backing field to cache the value and avoid addition string allocations.

Reply:
In this case there is an explicit binding mode set, so know knowledge of the defaults is needed: `{Binding ByOwnerOrAuthor, Mode=OneTime}`. This means the property will only be bound once.

I'm not sure why the string.Join was moved to this property originally, if it was intentional or just assumed to never be called more than once.